packages/lesmis/src/components/ProfileButton.js

```diff
@@ -18,15 +19,17 @@ const StyledProfileButton = styled(BaseProfileButton)`
 `;
 
 export const ProfileButton = () => {
-  const { data: user } = useUser();
+  const { data: user, isSuccess } = useUser();
   const queryClient = useQueryClient();
 
   const handleLogout = async () => {
     await post('logout');
     queryClient.invalidateQueries('user');
+    queryClient.invalidateQueries('entity');
+    queryClient.invalidateQueries('entities');
```

Comment:
Because it's used in two places quite far apart, perhaps there should be some place that this group of "queries to invalidate on auth change" is held. Either just a static constant with the queries listed out 
```
const QUERIES_INVALIDATED_ON_LOGIN_CHANGE = ['user', 'entity', 'entities'];
```
or a util function called something like  `invalidateQueriesOnLoginChange`

It may even be a smell that `handleLogout` should move to `mutations` so they're grouped together

Reply:
yeah good call, I added a useLogout mutation to capture the api logic in the api directory. I may add a list of constants later if the invalidation gets more complex.